src/config/src/config.rs

```diff
@@ -688,6 +687,8 @@ pub struct Limit {
     // MB, total data size in memory, default is 50% of system memory
     #[env_config(name = "ZO_MEM_TABLE_MAX_SIZE", default = 0)]
     pub mem_table_max_size: usize,
+    #[env_config(name = "ZO_MEM_TABLE_BUCKET_NUM", default = 0)] // default is 1
```

Comment:
nit: help string, as this is a new env var.

Reply:
Ha, this one is too simple, so i didn't add help

Reply:
added